index.js

```diff
@@ -43,6 +43,13 @@ load.path = function (dir) {
   var nearby = resolve(path.dirname(process.execPath))
   if (nearby) return nearby
 
+  // append platform, and prefix with scoped name matching package name if unscoped
+  var platformPackage = (packageName[0] === '@' ? '' : '@' + packageName + '/') + packageName + '-' + platform + '-' + arch
+  try {
+    var prebuildPackage = path.dirname(require('module').createRequire(path.join(dir, 'package.json')).resolve(platformPackage))
+    return resolveFile(prebuildPackage)
+  } catch(error) {}
```

Comment:
```suggestion
  } catch (error) {}
```

To please `standard`